dynamixel_motors: 
[
	{
		id: 1, 
		joint_name: rotation1_joint,
		interface_type: PosVel
	},
	{
		id: 2, 
		joint_name: shoulder1_joint,
		interface_type: PosVel
	},
	{
		id: 3, 
		joint_name: shoulder2_joint,
		interface_type: PosVel
	},
	{
		id: 4, 
		joint_name: rotation2_joint,
		interface_type: PosVel
	},
	{
		id: 5, 
		joint_name: shoulder3_joint,
		interface_type: PosVel
	},
	{
		id: 6, 
		joint_name: wrist_joint,
		interface_type: PosVel
	},
	{
		id: 7,
		joint_name: left_finger_joint,
		interface_type: PosVel
	},
	{
		id: 8,
		joint_name: right_finger_joint,
		interface_type: PosVel
	}
]

port_name: "/dev/USB2DYNAMIXEL"
port_baudrate: 1000000
